panel/custom.py

```diff
@@ -219,7 +248,7 @@ def _setup_autoreload(self):
         if not ((config.autoreload or getattr(self, '_debug', False)) and import_available('watchfiles')):
             return
         super()._setup_autoreload()
-        if (self._esm_path and not self._watching_esm):
+        if (self._esm_path(False) and not self._watching_esm):
```

Comment:
```suggestion
        if (self._esm_path(compile=False) and not self._watching_esm):
```

Nit, for readability.